assessment.md

```diff
@@ -143,39 +143,96 @@ TODO: add description for each assessment level for this skill.
 
 * Requirements engineering
 
-TODO: add description for each assessment level for this skill.
+Requirements engineering will be evaluated on this scale:
+
+* None = Fails to adds any requirements for the program and shows no knowledge of what the program is doing
+* Inadequate = Adds one requirement for the program
+* Adequate = Adds two requirements for the program
+* Good = Adds three requirements for the program
+* Excellent = Adds more than three requirements for the program and shows knowledge of what the program is doing
 
 * Software architecture
 
-TODO: add description for each assessment level for this skill.
+The architecture of the software will be evaluated on this scale:
+
+* None = The program demonstrates no understanding or implementation of software architecture.
+* Inadequate = The program displays experience with software architecture, but there are flow and design issues.
+* Adequate = The program displays experience and understanding of software architecture with minimal flow and design issues.
+* Good = the program displays an extensive knowledge of software architecture theory and underlying experience with no flow or design issues.
+* Excellent = the program displays an extensive knowledge of software theory and underlying experience as a means of conducting and managing the design/flow. Develops concepts and is able to help other team members with the architecture of their software.
 
 * Software design
 
-TODO: add description for each assessment level for this skill.
+Creating the best software blueprint to most efficiently solve a particular problem.
+
+* None = Doesn't even attempt to solve the problem.
+* Inadequate = Creates incorrect software to solve the problem.
+* Adequate = Creates correct software to solve the problem.
+* Good = Creates correct and efficient software to solve the problem.
+* Excellent = Creates correct and efficient software while constantly thinking of how to make it more efficient.
```

Comment:
I think that this skill involves being able to create a sound design for your project/program rather than being able to implement it effectively. Both of these aspects are important but this skill is specifically discussing design principles.

Reply:
This comment has not been addressed yet. Please make sure to add the suggested changes or clarify your implementation